src/Corrade/Utility/Debug.cpp

```diff
@@ -165,10 +219,11 @@ Error::Error(const Flags flags): Error{_globalErrorOutput, flags} {}
 
 Debug::~Debug() {
     if(_output && (_flags & InternalFlag::ValueWritten)) {
+#ifndef CORRADE_TARGET_WINDOWS
         /* Reset output color */
         if(_flags & InternalFlag::ColorWritten)
             *_output << "\033[0m";
-
+#endif
```

Comment:
And also no color resetting implicitly on Debug destruction (end of a line, for example). I think these are crucial features to avoid leaking state (broken console colors for subsequent commands).